client/client_webhook.py

```diff
@@ -48,7 +50,21 @@ def __init__(self, commit_data=None, api_url=None, pre_convert_bucket=None, cdn_
         else:
             self.source_url_base = None
 
+        # remove all files from previous process_webhook during testing
+        # files are available after run for audit
+        tempfile.tempdir = "/tmp"
+        shutil.rmtree(tempfile.tempdir, ignore_errors=True)
```

Comment:
With the above line, `tempfile.tempdir = "/tmp"`, this will try to remove all temp files and directories for all processes running on the system, not just those created by this script.